Refactored Apis with correct api-version

DheerendraRathor opened the pull request:
- Moved Operations Api to separate directory 2016-08-10 with examples
- Added itemName: Value in Operation Apis for x-ms-pageable
- Added x-ms-examples to Operations Api

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

DheerendraRathor committed:
Refactored Apis with correct api-version

- Moved Operations Api to separate directory 2016-08-10 with examples
- Added itemName: Value in Operation Apis for x-ms-pageable
- Added x-ms-examples to Operations Api
- Added license-header in readme and set it to MICROSOFT_MIT

olydis commented on arm-recoveryservicesbackup/2016-12-01/swagger/backupManagement.json:
```diff
@@ -18,35 +18,6 @@
     "application/json"
   ],
   "paths": {
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {
```
This path and method signature was different from the one you replaced it with (no `/Subscriptions/...`), which is a breaking change. Are your SDKs/CLIs released? Or will you bump up the version just because of this?

olydis replied:
seeing that you also fixed the `x-ms-pageable` thing (`itemName`), I assume this wasn't released yet?

DheerendraRathor replied:
Our SDK is currently in preview so we will be upgrading minor version for it. CLI is not yet released all it will be including these changes.

lmazuel commented on arm-recoveryservicesbackup/readme.md:
````diff
@@ -33,4 +35,15 @@ directive:
+
+```yaml
+python:
+  namespace: azure.mgmt.recoveryservices.backup
````
should be azure.mgmt.recoveryservicesbackup

DheerendraRathor replied:
Resolved

lmazuel commented on arm-recoveryservicesbackup/readme.md:
```diff
@@ -33,4 +35,15 @@ directive:
+python:
+  namespace: azure.mgmt.recoveryservices.backup
+  package-name: azure-mgmt-recoveryservicesbackup
+  package-version: 1.0.0-preview
```
Should be 0.1.0 or 1.0.0a1 to respect Python syntax:
https://www.python.org/dev/peps/pep-0440/#pre-releases

DheerendraRathor replied:
Resolved

lmazuel commented on arm-recoveryservicesbackup/readme.md:
````diff
@@ -33,4 +35,15 @@ directive:
+### Python
+
+```yaml
+python:
````
please override header to MICROSOFT_MIT_NO_VERSION for Python

DheerendraRathor replied:
Resolved.

olydis replied:
Nice, happy to see repro-able code gen settings being persisted here! 😃

DheerendraRathor committed:
Updated python code generation settings

- Set version to 0.1.0
- Changed namespace to azure.mgmt.recoveryservicesbackup
- Added MICROSOFT_MIT_NO_VERSION as license header

AutorestCI commented:
No modification for NodeJS

AutorestCI commented:
No modification for Python

AutorestCI commented:
No modification for Ruby
